dhcpv4/options_test.go

```diff
@@ -57,41 +42,29 @@ func TestOptionsFromBytes(t *testing.T) {
 		0, 0, 0, //padding
 	}
 	opts, err := OptionsFromBytes(options)
-	if err != nil {
-		t.Fatal(err)
-	}
-	// each padding byte counts as an option. Magic Cookie doesn't add up
-	if len(opts) != 5 {
-		t.Fatal("Invalid options length. Expected 5, got %v", len(opts))
-	}
-	if opts[0].Code != OptionNameServer {
-		t.Fatal("Invalid option code. Expected %v, got %v", OptionNameServer, opts[0].Code)
-	}
-	if !bytes.Equal(opts[0].Data, options[6:10]) {
-		t.Fatal("Invalid option data. Expected %v, got %v", options[6:10], opts[0].Data)
-	}
-	if opts[1].Code != OptionEnd {
-		t.Fatalf("Invalid option code. Expected %v, got %v", OptionEnd, opts[1].Code)
-	}
-	if opts[2].Code != OptionPad {
-		t.Fatalf("Invalid option code. Expected %v, got %v", OptionPad, opts[2].Code)
-	}
+	require.NoError(t, err)
+	require.Equal(t, []Option{
```

Comment:
I freaked out for a moment on this, then looked up the code, and under the hood it uses `reflect.DeepEqual` :)